ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/datacolumns/retriever/SimpleSidecarRetrieverTest.java

```diff
@@ -130,7 +130,7 @@ void sanityTest() {
     List<Blob> blobs = Stream.generate(dataStructureUtil::randomValidBlob).limit(1).toList();
     BeaconBlock block = blockResolver.addBlock(10, 1);
     List<DataColumnSidecar> sidecars =
-        miscHelpers.constructDataColumnSidecars(createSigned(block), blobs, kzg);
+        miscHelpers.constructDataColumnSidecarsOld(createSigned(block), blobs, kzg);
```

Comment:
Do we need this?

Reply:
I don't see other way to construct proofs for fixtures